gravity.sh

```diff
@@ -734,6 +739,9 @@ gravity_DownloadBlocklistFromUrl() {
       database_adlist_status "${adlistID}" "4"
     fi
   fi
+  # remove temp_file
+  rm "${patternBuffer}" > /dev/null 2>&1 || \
+    echo -e "  ${CROSS} Unable to remove ${patternBuffer}"
```

Comment:
Can do the same file handle sceme here for `patternBuffer` (we really need a style guide for variable names).